web/src/js/components/Header/FlowMenu.tsx

```diff
@@ -56,10 +68,14 @@ export default function FlowMenu(): JSX.Element {
                         <Button
                             title="[d]elete flow"
                             icon="fa-trash text-danger"
-                            onClick={() => dispatch(removeFlow(flow))}
+                            onClick={() => {
+                                dispatch(removeFlow(selectedFlows));
+                                dispatch(selectFlow([])); // clear flow selection
```

Comment:
I noticed a small bug: previously, when a user removed a flow, the next flow was automatically selected. Now, this no longer happens. It would be nice to keep that behaviour, what do you think?
This line seems to be the issue. I remember adding it because, without it, multiple deletions caused this API call:

`http://127.0.0.1:8081/flows/e394b6f1-7425-40ec-9387-c72d949feee6/request/content/Auto.json?lines=513
`

to fail with a **404 Not Found**. This happens because the flow ID is selected as the next flow even though it has already been deleted. After this failed call, we correctly call the endpoint again with the actual next flow, and everything works fine.
Now, I need to figure out how to solve that 😄

Reply:
Great catch! This one is important. Folks coming from the TUI will repeatedly press `d` to get rid of flows. 😄

Reply:
okay this is solved now